### PR TITLE
#162948810 Like or  dislike article

### PR DESCRIPTION
**Pull request by Georgeygigz**
**What does this PR do?**

* It creates functionality for liking or disliking an article.

**Description of Task to be completed?**

* Adding functionality for liking an article
* Adding functionality for disliking article

**How should this be manually tested?**

* clone this repo `https://github.com/andela/ah-django-unchained-frontend`
* `cd ah-django-unchained-frontend`
* `git checkout  ft-like-dislike-article-162948810`
* Install the required dependencies `npm install`
* `npm start to run the local server`

**Pivotal Tracker story**
[#162948804](https://www.pivotaltracker.com/story/show/162948810)

**Screenshot**
image

### CODE REVIEW

**Comment by cosmas28** [comment]
Fix merge conflicts.

File: src/pages/LikeDislikeArticle/index.jsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './LikeDislikeArticle.scss';
+
+export const LikeDislikeArticle = ({ like, dislike, article }) => {
+  return (
+    <div>
+      <button type="button" className="fa fa-thumbs-up fa-2x" onClick={like} />
+
```
Comment (cosmas28) [inline src/pages/LikeDislikeArticle/index.jsx]: remove this space

**Review by cosmas28 — CHANGES_REQUESTED** [review]

**Comment by cosmas28** [comment]
Kindly rebase and make sure your `NavBar` works like the way it is working on staging.

File: src/pages/LikeDislikeArticle/index.jsx (hunk 1 above)
Comment (Georgeygigz) [reply src/pages/LikeDislikeArticle/index.jsx]: Okay working on it

**Review by cosmas28 — APPROVED** [review]

**Pull request merged**